docs/BUILDING-BLOCKS.md

```diff
@@ -22,14 +22,14 @@ Concepts to grasp
 * [How we distribute sources to our users](DISTRIBUTION.md)
 
 Building blocks used right:
-* [Create dynamic resources for tables by reflecting a whole database](https://github.com/dlt-hub/verified-sources/blob/master/sources/sql_database/sql_database.py#L50)
+* [Create dynamic resources for tables by reflecting a whole database](https://github.com/dlt-hub/verified-sources/blob/master/sources/sql_database/__init__.py#L56)
 * [Incrementally dispatch github events to separate tables](https://github.com/dlt-hub/verified-sources/blob/master/sources/github/__init__.py#L70)
```

Comment:
to https://github.com/dlt-hub/verified-sources/blob/master/sources/github/__init__.py#L91-L95